bindings/pydeck/setup.py

```diff
@@ -209,6 +209,8 @@ def run(self):
         extras_require={
             "carto": ["pydeck_carto"],
             "jupyter": [
+                "ipywidgets>=7.0.0",
+                "traitlets>=4.3.2",
                 'ipython>=5.8.0;python_version<"3.4"',
                 'ipykernel>=5.1.2;python_version>="3.4"',
```

Comment:
It looks like the only place where we use `ipykernel` is in the tests:
https://github.com/visgl/deck.gl/blob/1049956d723fe7a92c39ea203bc2b5c5fe00cc36/bindings/pydeck/tests/conftest.py#L9

And the only place where we use `ipython` outside of tests is in
https://github.com/visgl/deck.gl/blob/017d960d95160c456204f781dceac41eafd49579/bindings/pydeck/pydeck/io/html.py#L102
and there we have handling so it works whether or not `ipython` exists, right? So do we need to have a runtime dep on that?